### PR TITLE
go: update to 1.16.5 (includes security updates)

### PR DESCRIPTION
**Pull request by fgma**
_Motivation:_  Latest minio release depends on go 1.16. This also fixes multiple security issues in go.
_Linked issues:_  -

Fixes multiple security issues: CVE-2021-3114, CVE-2021-3115, CVE-2021-27918, CVE-2021-31525, CVE-2021-27919, CVE-2021-27918, CVE-2021-33196, CVE-2021-33195, CVE-2021-33197

### Checklist
- [x] Build rule `all-supported` completed successfully
- [x] Package upgrade completed successfully
- [x] New installation of package completed successfully

### CODE REVIEW

**Comment by fgma** [comment]
Seems like fritzctl build is broken building with go 1.16 and the official repo is archived since april: https://github.com/bpicode/fritzctl

The last official release is from 2018 which does not include a go.mod. To make it build again we would at least need to patch the module handling. Any remarks on how to solve this?

**Comment by fgma** [comment]
@hgy59 Any feedback on this? I'd like to continue getting minio updated.

**Comment by hgy59** [comment]
> 
> 
> @hgy59 Any feedback on this? I'd like to continue getting minio updated.

I propose to remove fritzctl from synocli-net. As this is a very product specific tool this will be kept as diyspk only.

**Comment by fgma** [comment]
OK, so I will include the removal in this PR.

File: spk/synocli-net/Makefile [hunk 1]
```diff
@@ -1,24 +1,19 @@
 SPK_NAME = synocli-net
 SPK_VERS = 1.6
```
Comment (hgy59) [inline spk/synocli-net/Makefile:2]: IMHO we need to change the main version, as we have a breaking change (for users of fritzctrl).
SPK_VERS = 2.0

File: native/go/digests [hunk 2]
```diff
@@ -1,3 +1,3 @@
-go1.16.4.src.tar.gz SHA1 b1d5f9a63f0cda6580f6a31419af78dc0723536a
-go1.16.4.src.tar.gz SHA256 ae4f6b6e2a1677d31817984655a762074b5356da50fb58722b99104870d43503
-go1.16.4.src.tar.gz MD5 5884a4cb49067b9bcf6e69a1ef44259c
+go1.16.5.src.tar.gz SHA1 
+go1.16.5.src.tar.gz SHA256 
+go1.16.5.src.tar.gz MD5
```
Comment (hgy59) [inline native/go/digests]: use `make clean && make digests` to create valid digests file.

**Review by hgy59 — CHANGES_REQUESTED** [review]
some minor suggestions.

File: native/go/digests (hunk 2 above)
Comment (fgma) [reply native/go/digests]: Fixed in last commit.

File: spk/synocli-net/Makefile (hunk 1 above)
Comment (fgma) [reply spk/synocli-net/Makefile]: Fixed in last commit.

**Review by hgy59 — APPROVED** [review]
LGTM

**Pull request merged**